tutorials/surface_vehicles.md

```diff
@@ -28,8 +28,8 @@ To compile all the custom libraries in the right order `colcon` is recommended.
 The `colcon` tool is available on all platforms using `pip3`.
 
 ```bash
-mkdir ~/gazebo_maritime_ws
-wget https://raw.githubusercontent.com/gazebosim/gz-sim/gz-sim8/tutorials/files/surface_vehicles/gazebo_maritime_ws -o ~/gazebo_maritime_ws -r
+wget https://raw.githubusercontent.com/gazebosim/gz-sim/gz-sim8/tutorials/files/underwater_vehicles/gz_maritime_ws.zip -O ~/gz_maritime_ws.zip
```

Comment:
I think this comment https://github.com/gazebosim/gz-sim/pull/2330#pullrequestreview-1921387250 is talking about this line.

But it appears the file added is `tutorials/files/surface_vehicles/gz_maritime_ws.zip` (under `surface_vehicles`), whereas this line is referring to a zip file in `underwater_vehicles`? Is that a typo?

The individual files removed are in `surface_vehicles` here: https://github.com/gazebosim/gz-sim/tree/gz-sim8/tutorials/files/surface_vehicles
There's no zip file in `underwater_vehicles` right now, nor in this PR: https://github.com/gazebosim/gz-sim/tree/gz-sim8/tutorials/files/underwater_vehicles

Reply:
Oh, that was a typo, thanks! Fixed in [2d27c3f](https://github.com/gazebosim/gz-sim/pull/2330/commits/2d27c3fdb87689d70fda9d9ef762188b57065185).